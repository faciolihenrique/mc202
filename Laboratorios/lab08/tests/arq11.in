# Testes simples com três elementos
# Sem rotações
i 033799 Maria da Silva
i 031780 João de Souza
i 034144 José de Morais
p
a
d
l
# RR
i 031780 João de Souza
i 033799 Maria da Silva
i 034144 José de Morais
p 
a
d
l
# RL
i 031780 João de Souza
i 034144 José de Morais
i 033799 Maria da Silva
p
a
d
l
# LL
i 032210 Helena de Almeida
i 031780 João de Souza
i 013627 Marcos Pereira
p
a
d
l
# LR
i 032210 Helena de Almeida
i 013627 Marcos Pereira
i 031780 João de Souza
p
a
d
l
x
