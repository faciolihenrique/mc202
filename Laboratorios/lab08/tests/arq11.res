Ação: # Testes simples com três elementos
Ação: # Sem rotações
Ação: i 033799 Maria da Silva
Ação: i 031780 João de Souza
Ação: i 034144 José de Morais
Ação: p
031780 João de Souza
033799 Maria da Silva
034144 José de Morais
Ação: a
Altura da base de dados: 2.
Ação: d

   / 34144
  33799
   \ 31780

Ação: l
Ação: # RR
Ação: i 031780 João de Souza
Ação: i 033799 Maria da Silva
Ação: i 034144 José de Morais
Ação: p 
031780 João de Souza
033799 Maria da Silva
034144 José de Morais
Ação: a
Altura da base de dados: 2.
Ação: d

   / 34144
  33799
   \ 31780

Ação: l
Ação: # RL
Ação: i 031780 João de Souza
Ação: i 034144 José de Morais
Ação: i 033799 Maria da Silva
Ação: p
031780 João de Souza
033799 Maria da Silva
034144 José de Morais
Ação: a
Altura da base de dados: 2.
Ação: d

   / 34144
  33799
   \ 31780

Ação: l
Ação: # LL
Ação: i 032210 Helena de Almeida
Ação: i 031780 João de Souza
Ação: i 013627 Marcos Pereira
Ação: p
013627 Marcos Pereira
031780 João de Souza
032210 Helena de Almeida
Ação: a
Altura da base de dados: 2.
Ação: d

   / 32210
  31780
   \ 13627

Ação: l
Ação: # LR
Ação: i 032210 Helena de Almeida
Ação: i 013627 Marcos Pereira
Ação: i 031780 João de Souza
Ação: p
013627 Marcos Pereira
031780 João de Souza
032210 Helena de Almeida
Ação: a
Altura da base de dados: 2.
Ação: d

   / 32210
  31780
   \ 13627

Ação: l
Ação: x
Processamento terminado
