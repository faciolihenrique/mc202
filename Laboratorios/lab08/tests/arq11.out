Ação: # Testes simples com três elementos
Ação: # Sem rotações
Ação: i 033799 Maria da Silva
Ação: i 031780 João de Souza
Ação: i 034144 José de Morais
Ação: p
031780 João de Souza
033799 Maria da Silva
034144 José de Morais
Ação: a
Altura da base de dados: 2.
Ação: d

   / 34144
  33799
   \ 31780

Ação: l
Ação: # RR
Ação: i 031780 João de Souza
Ação: i 033799 Maria da Silva
Ação: i 034144 José de Morais
